src/main.lua

```diff
@@ -867,9 +864,7 @@ addEventingHandler(ActionMap.IncreaseVault, utils.hasMatchingTag("Action", Actio
 	})
 end)
 
-addEventingHandler(ActionMap.BuyRecord, function(msg)
-	return msg.Action == ActionMap.BuyRecord or msg.Action == ActionMap.BuyName
-end, function(msg)
+addEventingHandler(ActionMap.BuyName, utils.hasMatchingTag("Action", ActionMap.BuyName), function(msg)
```

Comment:
SDK v3+ uses `Buy-Name`

Reply:
Will the handler named "Buy-Record" be removed now or will you remove it manually?

Reply:
i'll remove it testnet - it doesn't cleanup itself 😢